libvast/src/command.cpp

```diff
@@ -69,7 +80,7 @@ caf::message run(const command& cmd, caf::actor_system& sys,
   bool has_subcommand;
   switch(state) {
     default:
-      return make_message(make_error(ec::unrecognized_option, *position));
+      return help_and_make_error_msg(cmd, ec::unrecognized_option, *position);
```

Comment:
`return make_help_and_error_msg(...)` reads a bit clearer than a combination of a noun ("help") and an action ("make_error_msg"). Alternatively,  we could use `return make_error_msg`; I'm not sure if the side-effect is worth encoding into the function name.